packages/slate-editor/src/hyperscript.ts

```diff
@@ -122,7 +123,9 @@ const extensions = [
 ];
 
 const creators: Record<keyof HElements, HyperscriptCreators[string]> = {
-    editor: createEditorFactory(() => createEditor(createSlateEditor(), () => extensions)),
+    editor: createEditorFactory(() =>
+        createEditor(createSlateEditor(), () => extensions, [withReact, withHistory]),
+    ),
```

Comment:
Hm... Isn't it weird that the tests hyperscript is not using the Plate's editor implementation? :thinking:

Reply:
A little bit, yes. It is on my roadmap to rework the tests to use Plate (soon). It will be required when we start refactoring plugins/extensions to Plate plugins as otherwise we can't use them in tests.